source/NVDAObjects/UIA/winConsoleUIA.py

```diff
@@ -235,7 +235,7 @@ class WinConsoleUIA(KeyboardHandlerBasedTypedCharSupport):
 	#: Only process text changes every 30 ms, in case the console is getting
 	#: a lot of text.
 	STABILIZE_DELAY = 0.03
-	_TextInfo = consoleUIATextInfo
+	TextInfo = consoleUIATextInfo
```

Comment:
Please add a comment on this. If I understand correctly: 

Overriding `_get_TextInfo` and thus the `TextInfo` property on `NVDAObjects.UIA.UIA`
consoleUIATextInfo works around issues with collapsed UIA text ranges.

Reply:
I think it's key to use the term "overriding" in this comment, also very helpful for these dynamically built inheritance hierarchies if you refer to the thing you are expecting to override. After all, the logic of this is tightly coupled to that.

Thinking about this some more, given that you are overriding _get_TextInfo I think it makes sense to define it here in the same way. That way, when people search for _get_TextInfo looking for implementations, they find this one.

When this class is used, is it always on top of `NVDAObjects.UIA.UIA`? If not then the below comment should be adjust to mention what it IS always overriding. 

```suggestion
	# Overriding _get_TextInfo and thus the TextInfo property on NVDAObjects.UIA.UIA
	# consoleUIATextInfo fixes expand/collapse, implements word movement, and 
	# bounds review to the visible text.
	def _get_TextInfo(self):
		return consoleUIATextInfo
```